src/tests/eet/eet_suite.c

```diff
@@ -8,6 +8,14 @@
 #include "../efl_check.h"
 #include <Eet.h>
 
+#ifdef _MSC_VER
+/* On windows, applink.c needs to be included in the executable; otherwise
+ * all the tests which are using file pointer will fail in runtime, throwing
+ * `No OPENSSL_Applink` messages, i.e: eet_test_identity.
+ * */
+# include <openssl/applink.c>
```

Comment:
Opened the code crossing my fingers that we didn't need applink.

Guess I was wrong. Oh well!

Reply:
hahah yeah, I tried my best with no success removing it...